.github/workflows/linting.yml

```diff
@@ -761,10 +761,16 @@ jobs:
           extensions: mysql, imagick
           coverage: none
 
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ steps.versions.outputs.node-version }}
```

Comment:
What's going on with all these additions? That suggests to me that your change is running extra code that it probably shouldn't.

Reply:
I was getting test failures that `pnpx` was missing. I can't remember now what those were - I will try removing these changes to see what breaks.

Reply:
I suspect that you're right, it was running unnecessary code, and now I made some other changes `pnpx` might not be necessary.